contextily/tile.py

```diff
@@ -39,9 +40,8 @@ def _clear_cache():
 atexit.register(_clear_cache)
 
 
-def bounds2raster(
-    w, s, e, n, path, zoom="auto", url=None, ll=False, wait=0, max_retries=2
-):
+def bounds2raster(w, s, e, n, path, zoom="auto", url=None, source=None,
```

Comment:
I think we can move `url` to the end of the signature (and also in the docs), to put less attention to it. 
Since source just replaces it, the order stays the same, so that should be OK in case people were using all positional arguments

Reply:
smart, have done!